rapidsai/devel-rocky.arm64.Dockerfile

```diff
@@ -1,6 +1,6 @@
-ARG FROM_IMAGE=gpuci/miniforge-cuda-arm64
-ARG CUDA_VER=11.2
-ARG LINUX_VER=centos8
+ARG FROM_IMAGE=nvidia/cuda
+ARG CUDA_VER=11.6.1
```

Comment:
We don't support CUDA 11.6, so this shouldn't change.

Reply:
changed to 11.5